python/GafferOFXUITest/DocumentationTest.py

```
@@ -0,0 +1,54 @@
+#
+##########################################################################
+
+import GafferUITest
+
+import GafferImage
+import GafferOFX
+import GafferOFXUI
+
+class DocumentationTest( GafferUITest.TestCase ) :
+
+	def test( self ) :
```

Ruff (0.8.2)

Missing return type annotation for public function `test`

Add return type annotation: `None`

(ANN201)